nilearn/masking.py

```diff
@@ -126,6 +125,8 @@ def intersect_masks(mask_imgs, threshold=0.5, connected=True):
         grp_mask: 3D nibabel.Nifti1Image
             intersection of all masks.
     """
+    from .image import new_img_like
```

Comment:
I did this because there appeared to be many circular imports when directly importing at the topic in the lower level scripts.